src/Microsoft.Health.Functions.Extensions/AzureFunctionsJobHostBuilder.cs

```diff
@@ -56,6 +60,20 @@ public AzureFunctionsJobHostBuilder ConfigureWebJobs(Action<IWebJobsBuilder> con
         _configureWebJobs += (c, b, l) => configure(b);
         return this;
     }
+    internal class NullTelemetryChannel : ITelemetryChannel
```

Comment:
If this is going to be defined in the same class, let's make this class `private` and `sealed`

Reply:
Since this isn't used outside of the type